### ISSUE TITLE
Cannot install with Terraform 0.13

### ISSUE THREAD

**Issue by rayrutjes** [louy/terraform-provider-uptimerobot#80]
Using the following configuration with Terraform 0.13:

```hcl
terraform {
  required_providers {
    uptimerobot = {
      source = "louy/uptimerobot"
      version = "0.5.0"
    }
  }
  required_version = ">= 0.13"
}

```

Getting the following error: 

```
Error: Failed to query available provider packages

Could not retrieve the list of available versions for provider -/uptimerobot:
provider registry registry.terraform.io does not have a provider named
registry.terraform.io/-/uptimerobot


Error: Failed to validate installed provider

Validating provider louy/uptimerobot v0.5.0 failed: provider binary not found:
could not find executable file starting with terraform-provider-uptimerobot
```

I followed the official site recommendations:
![image](https://user-images.githubusercontent.com/5570853/92331066-0fca2500-f074-11ea-8a03-e3e5781f306f.png)

**Comment by louy** [comment]
damn it sorry should've tested this properly. I'll take a look

**Comment by Kissy** [comment]
It seems that the issue comes from the new signature in terraform v0.13.
https://www.terraform.io/docs/plugins/signing.html

**Comment by Elindorath** [comment]
@Kissy Where do you find the executable?

**Comment by collingo** [comment]
Has there been any progress on this one? I am getting the following when trying to `init` with tf `v0.13.2`. Slightly different message this time but might be related...
```
Error: Failed to install provider

Error while installing louy/uptimerobot v0.5.0: zip: not a valid zip file
```

PS @Elindorath in case you haven't heard from Kissy yet, you can download the binary from the release page - https://github.com/louy/terraform-provider-uptimerobot/releases/tag/v0.5.0

**Comment by louy** [comment]
As @collingo mentioned this seems to be an issue with terraform itself. You can find more details here: https://github.com/hashicorp/terraform/issues/26282

Issue fixed in v0.5.1
